flir_ptu_driver/src/driver.cpp

```diff
@@ -67,6 +67,14 @@ bool PTU::initialized()
   return !!ser_ && ser_->isOpen() && initialized_;
 }
 
+bool PTU::disable_limits()
+{
+  ser_->write("ld ");  // Disable Limits
+  ser_->read(20);
+  Lim = false;
+  return true;
```

Comment:
Is the twenty byte response supposed to be a confirmation or at least an echo? Should we be checking that at all before returning true?

Reply:
Not really, mostly read to clear buffer, else crash on next publish.
Likelihood of this failing is very low, not done for position commands originally either.